package.json

```diff
@@ -6,7 +6,7 @@
     "@cosmotech/api-ts": "^3.0.1-SNAPSHOT",
     "@cosmotech/azure": "^1.3.4",
     "@cosmotech/core": "^1.14.0",
-    "@cosmotech/ui": "^7.0.0",
+    "@cosmotech/ui": "file:.yalc/@cosmotech/ui",
```

Comment:
This change is related to the local link made with yalc. Could you remove this file from the commit?
Same thing for the file _yarn.lock_

Reply:
I knew it, thanks!